### DISCUSSION TITLE
Waters DIA

### DISCUSSION THREAD

**Discussion by cbroeckl** [vdemichev/DiaNN#101] [General]
I just ran across the nice recent nature biotech paper: https://www.nature.com/articles/s41587-021-00860-4.  Really impressive and exciting, and clearly the software plays a critical role here.   I have a few questions before i begin to dive into DIA-NN:

1. Scanning SWATH seems to be essentially identical to SONAR from waters.  Is it safe to assume that if i have SONAR proteomics data on a comparable platform that that used in the nature biotech paper that it could be processed with DIA-NN?   
2. Additionally, SONAR and HDMSe have essentially the same data structure, but rather than a quad-based precursor selection with SONAR we have ion mobility based precursor selection with HDMSe - would DIA-NN be able process HDMSe data? 
3. Finally - are there components of the DIA-NN algorithm that are specifically applicable to peptides/proteins, or would you expect it to work with small molecules, lipids, glycans, etc?   

General questions, but i do intend to dive into this program in the near future if i can generate data in my lab that DIA-NN can handle (Waters SONAR and/or HDMSe).  Thanks!
Corey

**Comment by vdemichev** [comment]
Hi Corey,

Thanks for appreciating the paper and your interest in DIA-NN!))

About Waters data: we have never worked with it ourselves. However the Scanning SWATH principle is indeed similar to that of SONAR (although there are probably lots of differences in the technical implementation), and I heard from a collaborator that DIA-NN does seem to work with SONAR data once the latter is converted to .mzML. So with any other data type I will just try it after conversion to .mzML, there's a chance it might work.

Decoy peptide generation is specific to proteomics. DIA-NN might work with metabolomic/lipidomic data, if there's a spectral library provided that is supplied with decoys. But this has never been validated, so no guarantees here. 

Best wishes,

Vadim

**Reply by cbroeckl** [reply]
Thanks Vadim,
I am more interested in the preprocessing steps at this point - MS1 level peak detection and fragment ion assignment.  From your description it sounds as though that should be fairly agnostic to molecule type - there are no peptide centric rules built into the program such as amino acid residue masses, charge state, etc?   

I will try to generate some SONAR proteome data and give it a try.  i think I can dig up some HDMSe data somewhere for testing.   thanks.
Corey

**Reply by vdemichev** [reply]
DIA-NN does not really do MS1 peak detection, it basically extracts all chromatograms - for the precursor itself (MS1) and its fragments (MS2) - and tries to find the retention time at which the signal across these seems most consistent. But for assessing the confidence of the resulting "IDs" it does need decoys. There's also an option to use the 'second best' peak for each precursor instead of decoys (--second-best option in DIA-NN), but this is not particularly stable. So in theory it can work for metabolomics, but I've never tried it. 

Vadim

**Reply by cbroeckl** [reply]
Thanks for clarifying - is it accurate to say that all quantification is based on MS2 chromatograms then?   

assessing identification confidence in metabolomics is a currently unresolved issue - decoy databases are rather problematic, so the community is still working through the most appropriate way to define ID FDR.   I do not expect DIA-NN to resolve this issue!  
thanks again.
Corey

**Reply by vdemichev** [reply]
There's MS1 quant too (Ms1.Area column in the main report - it's non-normalised, i.e. direct integration of the MS1 profile).

**Reply by cbroeckl** [reply]
And it appears that when processing a collection of injections, that signals are aligned across samples, correct?  i.e. i should be able to get a rectangular matrix of non-normalized MS1 and/or MS2 signal intensities?

**Reply by vdemichev** [reply]
Alignment will only work for proteomics, unfortunately.

**Reply by cbroeckl** [reply]
Thanks - my primary goal is to use it for proteomics, so that will be fine.

**Comment by cbroeckl** [comment]
Vadim,

I downloaded the .wiff files from the proteomeExchange site and noted that in the .wiff format (as viewed in seeMS), the precursor mass is labelled with a mass center, and the isolation window is labelled with a mass range.   What is puzzling is that the reported isolation window is inconsistent with the file names - i.e. the file named '20200131_yeast_sswath_ct05_10da_20ul_3_Proc51f.wiff' is reported to have an isolation width of 2 Da, rather than the reported 10Da.   This suggests that DIA-NN doesn't actually use the isolation width in the algorithm - is this accurate?   Does it only use the 'Experiment' value or the precursor mass center?   Is the isolation width set elsewhere?  

Also - is there a vignette describing how to use DIA-NN for scanning swath/sonar experiments?  I was able to walk through the DIA vignette on the github readme successfullly, and am going through the output to familiarize myself with what 'success' looks like .  Thanks again, not only for the software but the prompt responses to all the questions/inquiries here. 

Corey

**Reply by vdemichev** [reply]
Hi Corey,

The way Scanning SWATH data is written to .wiff files depends on the acquisition software settings and does not really reflect how the data was acquired by the instrument. Basically, the data here was 'binned' in windows of 2Da, while the real Q1 window size was 10Da. With such increased 'resolution' in the Q1 dimension we are able to extract this Scanning SWATH-specific information about the precursor mass of each fragment trace we see in the data. DIA-NN was co-developed with Scanning SWATH, so it supports it fully.

There's essentially no difference in DIA-NN settings for Scanning SWATH or other types of DIA data. One thing to keep in mind is that for Sciex TripleTOF the optimal MS2 tolerance is 20 ppm in most cases.

Best,
Vadim

**Reply by cbroeckl** [reply]
Thanks Vadim - i am interested in implementing this using Waters Sonar, and would like to determine whether the default output format will be sufficient.  you mentioned earlier that you had heard from another group that was successful in processing sonar data - it would be wonderful if you could put me in contact with that person so i can ensure the best chance of success as we head down this road.   

Specifically, i have noticed that the default waters output does not have mass bins labelled with a precursor mass or width, just 'bin' number.  I think i can find a way to fix this via inferrence of m/z center by bin, but it would be great to talk to someone who has already been down the path.   thanks again.

**Reply by vdemichev** [reply]
They way I heard it is that simply converting Waters SONAR to .mzML with MSConvert and then analysing with DIA-NN works without further tweaks. Have you tried it?

Vadim

**Reply by cbroeckl** [reply]
i haven't - i assumed that since the MS header info for the MS2 scans are different there would be a problem, but maybe i am overthinking it!

**Reply by HansVissers** [reply]
Dear Vadim and Corey, A new format and mzML CV terms have been introduced recently for scanning quadrupole DIA applications such as SONAR. If you would be interested in evaluating and implementing this I would encourage you to get in contact with the ProteoWizard and Skyline development teams. Regards, Hans

**Reply by vdemichev** [reply]
Hi Hans, thank you for the info!

**Reply by cbroeckl** [reply]
Thanks Hans - i will give this a try!  I have an injection of HeLa using SONAR i can try it with.
